### DISCUSSION TITLE
Why do main filters go into the custom list?

### DISCUSSION THREAD

**Discussion by ddday** [uBlockOrigin/uBlock-issues#2528] [Q&A] [answered]
Just wondering why filters will sometimes be removed from the main list and go into the custom section. Are they deprecated?

**Comment by gothic-bum** [comment, marked as answer]
Yes, when filter lists are removed as defaults they are moved to the custom filters section.  
  
And on a somewhat related note, please see the following links regarding "bad lists":  
[Add support for blocklist of filter lists](https://github.com/gorhill/uBlock/commit/23f08f027460b5d0c0436357f866d6c48b8df701)    
[Add concept of "really bad list" to badlists infrastructure](https://github.com/gorhill/uBlock/commit/4150c17f4a6e1b3bd0c68b91da74c358a92c03ad)

**Reply by ddday** [reply]
Thanks for the reply. So are these lists removed because they're considered "bad" after an update as referenced above, or are they just no longer maintained? ( I did see that as one of the qualifiers of a "bad" list, just want some clarification )

**Reply by gothic-bum** [reply]
It could be for many reasons, such as lack of maintenance, incompatibility with uBO, conflicts with other lists, etc.  
  
Examples:
https://github.com/gorhill/uBlock/commit/858fdaced3fbaf2d4c16aaa71cdab2b050e890b7
https://github.com/gorhill/uBlock/commit/98c0bd840678d322475f114f5c212c4ce63a6f05
https://github.com/gorhill/uBlock/commit/9f693c75c7e59231c5397cdccefb01bb27847c53

**Reply by ddday** [reply]
Cool, thanks!
